lib/v2/screens/dashboard/components/transactions_widget.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/transactions_bloc.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/transactions_events.dart';
+
+class TransactionsWidget extends StatefulWidget {
```

Comment:
This has changed and now it does... we need the state keeper mixin for all these otherwise they keep reloading when the user scrolls around the screen. 

Basically the AutomaticKeepAliveClientMixin makes all these components behave like we want them to.